boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -218,10 +218,98 @@ However, we need to short 2 pins on Host Connector J3 via a 3.3k resistor
 (simple shorting without the resistor will also do) for debugging to work.
 Those 2 pins are Pin5 HOST_RST_N_LT_R) and Pin21 (+V_HOST_3P3_1P8).
 
+MCUbootloader Support
+=====================
+
+Intel S1000 on Zephyr supports the open source, cross-RTOS MCUbootloader.
+MCUbootloader in conjuntion with USB DFU (Device Firmware Upgrade)
+over the air can be used for Field upgradation of Intel S1000 Firmware.
```

Comment:
USB DFU is not exactly over the "air", please remove "over the air". Also change "upgradation" to "upgrades".

Reply:
Will this procedure (and upgrading the S1000 firmware) work on the board referenced at the top of this doc (https://software.intel.com/en-us/iot/speech-enabling-dev-kit)?  I suspect not, and we need to be more explicit at the top of this doc about the board and procedures being documented here.

Reply:
@dbkinder This procedure will work on the board referenced at the top of this doc.